build/scripts/build.ps1

```diff
@@ -102,6 +102,7 @@ function Process-Arguments() {
     if ($cibuild -and -not $official -and $anyVsi) {
         # Avoid spending time in analyzers when requested, and also in the slowest integration test builds
         $script:skipAnalyzers = $true
+        $script:bootstrap = $false
```

Comment:
💭 I wouldn't complain about a corresponding update to the preceding comment

Reply:
I have a followup PR for this. Will update the comment. 

Using a follow up PR cause I want to get this merged and get us out of the intermmediate state we're in ATM.